Package.swift

```diff
@@ -816,6 +829,58 @@ let package = Package(
       ]
     ),
 
+    .target(
+      name: "FirebasePerformance",
+      dependencies: [
+        "FirebaseCore",
+        "FirebaseInstallations",
+        "FirebaseRemoteConfig",
+        .product(name: "GoogleDataTransport", package: "GoogleDataTransport"),
+        .product(name: "GULEnvironment", package: "GoogleUtilities"),
+        .product(name: "GULISASwizzler", package: "GoogleUtilities"),
+        .product(name: "GULMethodSwizzler", package: "GoogleUtilities"),
+        .product(name: "GULUserDefaults", package: "GoogleUtilities"),
+        .product(name: "GULSwizzlerTestHelpers", package: "GoogleUtilities"),
+        .product(name: "nanopb", package: "nanopb"),
+        .product(name: "GULSwizzlerTestHelpers", package: "GoogleUtilities"),
+      ],
+      path: "FirebasePerformance/Sources",
+      publicHeadersPath: "Public",
+      cSettings: [
+        .headerSearchPath("../../"),
+        .define("PB_FIELD_32BIT", to: "1"),
+        .define("PB_NO_PACKED_STRUCTS", to: "1"),
+        .define("PB_ENABLE_MALLOC", to: "1"),
+        .define("FIRPerformance_LIB_VERSION", to: firebaseVersion),
```

Comment:
Not related to this PR, but `FIRPerformance_LIB_VERSION` should be eliminated and use the FirebaseCore public API to get the version instead - https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore/Sources/Public/FirebaseCore/FIRVersion.h#L22

Reply:
Noted. Will address this in another PR.